addons/mass_mailing/models/mail_mail.py

```diff
@@ -21,7 +21,7 @@ def create(self, values_list):
         mails = super(MailMail, self).create(values_list)
         for mail, values in zip(mails, values_list):
             if values.get('mailing_trace_ids'):
-                mail.mailing_trace_ids.write({'message_id': mail.message_id})
+                mail.mailing_trace_ids.write({'message_id': mail.message_id, 'email': mail.email_to if mail.email_to else mail.mailing_trace_ids.email})
```

Comment:
Trace already contain the email normally ? Or is there an issue with it ?

Reply:
Yes, there was an issue with it, `email` field was not correct with multiple email